### DISCUSSION TITLE
Exported top-down model not working in bonsai

### DISCUSSION THREAD

**Discussion by MForsthofer** [talmolab/sleap#1626] [Help!]
I'm currently trying to track animals using sleap within bonsai. This works for single instance, but not for top-down models, where bonsai throws the error 'Object reference not set to an instance of an object' . I'm not entirely sure if this is caused by how i export the model or by the bonsai node (PredictPoses), so i wanted to double check. (Should also note, the model works well to track within the GUI). 

To export, i export the centroid and the centered instance model into one frozen graph, using the command 

sleap-export --model "[path to centroid model]" --model "[path to centered instance model]" --export_path "[path to centered instance model]" --max_instances "1". 

I've also attached my training configs, would be happy for any kind of help.

Centroid config: 
[training_config_centroid.json](https://github.com/talmolab/sleap/files/13589987/training_config_centroid.json)

Centered config:
[training_config_centeredinstance.json](https://github.com/talmolab/sleap/files/13589978/training_config_centeredinstance.json)

**Comment by talmo** [comment]
Hi @MForsthofer,

Yep, we're aware of this issue and have someone actively working to address it. Feel free to follow along in #1542.

Sorry about that!

Talmo
